src/client/encrypt.js

```diff
@@ -24,6 +24,7 @@ module.exports = function (client, options) {
       } else {
         if (packet.serverId !== '-') {
```

Comment:
btw looking at the code, what's the point of this check? so the response is sent anyway below...

Reply:
What is the purpose of your PR ? What does this do?

Reply:
Should be added to the doc

Reply:
btw it was inspired by the pakkit patch, in my case it was also useful to have in the web app because I was not even getting an answer back from the server, so I was using this event to display an error message to the user

Reply:
It's not generally a good idea to make a change to code without understanding what's actually happening.

The creation of this event implies there is actually some credential issue here, can you look into what's happening here?

Reply:
> It's not generally a good idea to make a change to code without understanding what's actually happening.

Yes, you are right, I just saw the message in the debug console and decided to also put an event there. But doesn't it mean what it says?

> The creation of this event implies there is actually some credential issue here, can you look into what's happening here?

WDYM?
Do you mean there is an issue with my application instead?

Reply:
My goal is to end the connection as soon as I understand that the server is running online mode